axum/src/extract/path/mod.rs

```diff
@@ -347,6 +357,11 @@ impl fmt::Display for ErrorKind {
                 f,
                 "Cannot parse value at index {index} with value `{value:?}` to a `{expected_type}`"
             ),
+            ErrorKind::DeserializeError {
+                key,
+                value,
+                message,
+            } => write!(f, "Cannot parse `{key}` with value `{value:?}`: {message}"),
```

Comment:
Do not use debug for `value`, it would add redundant quotes. You can also that in the new tests.

Reply:
I was just mirroring the practice established by the other formats for the other variants: https://github.com/tokio-rs/axum/blob/e3bb7083c886247f4e6931e149ef6067e6b82e1b/axum/src/extract/path/mod.rs#L330-L349

I was a bit perplexed about this as well, and thought it did not make sense to use debug format for `value`.

Should I change it do non-Debug for this new variant only, or change the other variants as well?

Reply:
I added an additional commit that changes the format for all variants.

Reply:
Thanks. Sorry I didn't notice you just wrote it consistently with the rest, otherwise I wouldn't bring it up here.